spec/api/create_client_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rspec'
+require_relative '../support/data'
```

Comment:
I believe you can require files in spec_helper.rb so their modules and methods should be visible in all _spec.rb files.